psydac/core/bsplines.py

```diff
@@ -266,7 +271,8 @@ def basis_funs_all_ders(knots, degree, x, span, n, normalization='B', out=None):
         ders[i,j] = (d/dx)^i B_k(x) with k=(span-degree+j),
         for 0 <= i <= n and 0 <= j <= degree+1.
     """
-
+    knots = np.asarray(knots)
```

Comment:
As above

Reply:
Done in https://github.com/pyccel/psydac/commit/54fa1fb4c2afba79c77392308a193bad59161fee